Week-4/Javascript_basics/this_keyword/script.js

```diff
@@ -0,0 +1,32 @@
+class Person {
+  constructor(name, age, gender, nationality) {
+    this.Name = name;
+    this.Age = age;
+    this.Gender = gender;
+    this.Nationality = nationality;
```

Comment:
why are these capitalised on LHS?

Reply:
i didn't want the input and property to have the exact same characters

Reply:
it can have same, as we're setting values to this instance.
if you had to do :
x = x
here you would need refactor on LHS
but 
this.x = x 
this is legal.